src/docs/devices/TagTuner/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: TagTuner
+date-published: 2024-10-27
+type: misc
+standard: global
+board: esp32
+project-url: https://luka6000.github.io/TagTuner
+made-for-esphome: True
```

Comment:
@jesserockz is this device in the made for esphome program? just confirming.

Reply:
Still pending approval =)
(I made some labels to help)